distribution/changelog.txt

```diff
@@ -3,6 +3,7 @@
 - Feature: [#7956, #7964] Add sprite font glyphs for Hungarian and some Czech letters.
 - Improved: [#7930] Automatically create folders for custom content.
 - Removed: [#7929] Support for scenario text objects.
+- Fix: [#2940] Fixed so that the staff patrol areas can be mouse-painted.
```

Comment:
I think this is more of an improvement than a fix.

` - Improved: [#2940] Allow dragging to set patrol area`

The list is also ordered by issue number, so this should be moved up to line 4.